src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/header/regexpheader/Example3.txt

```diff
@@ -0,0 +1,15 @@
+/*xml
+<module name="Checker">
+  <module name="RegexpHeader">
+    <property
+      name="header"
+      value="^// Copyright \(C\) (\d\d\d\d -)? 2004 MyCompany$
+        \n^// All rights reserved$"/>
+  </module>
+</module>
+*/
+
+// xdoc section -- start
+^// Copyright \(C\) (\d\d\d\d -)? 2004 MyCompany$
+^// All rights reserved$
```

Comment:
It is not obvious to me how this is translated to https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/620eec4_2023152225/checks/header/header.html#Example3-code.

Reply:
It is converted to https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/620eec4_2023152225/checks/header/regexpheader.html#Example3-code

You looked at different Check

Reply:
Ah, I see, thanks for pointing this out :)

Reply:
https://github.com/checkstyle/checkstyle/issues/13585